querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/ExtendedBeanSerializer.java

```diff
@@ -30,7 +30,7 @@
 import com.querydsl.sql.codegen.support.PrimaryKeyData;
 
 /**
- * ExtendedBeanSerialzier outputs primary key based equals, hashCode and toString implementations
+ * ExtendedBeanSerializer outputs primary key based equals, hashCode and toString implementations
```

Comment:
Preferably another PR. Let's try to get some merges done also to make new work easier.